pkg/virt-operator/kubevirt_test.go

```diff
@@ -994,6 +1048,57 @@ var _ = Describe("KubeVirt Operator", func() {
 
 		}, 15)
 
+		It("delete temporary validation webhook once virt-api is deployed", func(done Done) {
```

Comment:
Did I miss the test which ensures that the dummy webhook gets created on new CRs?

Reply:
it's in there.

look at ```It("should add resources on create", func(done Done) {```

We check that a validation webhook entry is generated on create before api-server is rolled over. at the bottom of the function, we're checking that the validation webhook is in the cache.